### PR TITLE
Gal 606/render clinic fill target and radius

### PR DESCRIPTION
**Pull request by msandhuNHS**
## Description

Updated frontend to fetch clinic data values from db.

1. Rendering fillToTargetPercentage
2. Rendering lastSelectedRange
3. Removed console logs
4. Refactoring clinic information component code

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: src/app/views/clinic_information/ClinicInformation.js [hunk 1]
```diff
@@ -280,6 +286,7 @@ class ClinicInformation extends Component {
 
         let initialSelectedClinicId = response.data[0].ClinicId.S
         let initialSelectedClinic = response.data[0].ClinicName.S
+        console.log(initialSelectedClinic);
```
Comment (zainm97) [inline src/app/views/clinic_information/ClinicInformation.js:289]: left in log

File: src/app/components/pagination.js [hunk 2]
```diff
@@ -42,11 +42,11 @@ export default class Pagination extends Component {
   }
 
   onPrevHandler(e) {
-    console.log('prev');
+    // console.log('prev');
```
Comment (zainm97) [inline src/app/components/pagination.js:45]: this looks incomplete

File: src/app/components/pagination.js [hunk 3]
```diff
@@ -42,11 +42,11 @@ export default class Pagination extends Component {
...
   }
 
   onNextHandler(e) {
-    console.log('next');
+    // console.log('next');
```
Comment (zainm97) [inline src/app/components/pagination.js:49]: this looks incomplete

File: gulpfile.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ function compileStyles() {
     .pipe(sass())
     .pipe(gulp.dest('src/app/styles/css/'))
     .on('error', (err) => {
-      console.log(err)
+      // console.log(err)
```
Comment (zainm97) [inline gulpfile.js:15]: do you need to comment this back in? Need to use console.error then wont need process.exit(1) below

File: src/app/views/clinic_information/ClinicInformation.js [hunk 5]
```diff
@@ -315,6 +322,9 @@ class ClinicInformation extends Component {
             const [firstWordAfterComma] = (addressParts[1].trim()).split(' ');
             const displayViewAllPrevInvitations = prevInviteDate ? true : false;
 
+            const lastSelectedRange = response.data.LastSelectedRange.N;
```
Comment (zainm97) [inline src/app/views/clinic_information/ClinicInformation.js:325]: theres a lot of duplicated code. Seems like a good opportunity to store code in single place and call it here

File: src/app/views/clinic_information/ClinicInformation.js
Comment (zainm97) [inline src/app/views/clinic_information/ClinicInformation.js:1]: You haven't pulled on to main. You need to update the correct axios calls to pass through these values. And then update the "miles" in the axios call to the get-lsoa-in-range to pass through this value on default

File: src/app/views/invitation_planning/InvitationPlanning.js [hunk 6]
```diff
@@ -49,7 +49,7 @@ class InvitationPlanning extends Component {
         { forecastUptake: Number(value) }
       )
       .then((response) => {
-        console.log("response -> " + response.status);
+
```
Comment (zainm97) [inline src/app/views/invitation_planning/InvitationPlanning.js:52]: ?

**Review by zainm97 — CHANGES_REQUESTED** [review]
Please address changes

File: src/app/views/clinic_information/ClinicInformation.js
Comment (msandhuNHS) [reply src/app/views/clinic_information/ClinicInformation.js:1]: That may be another tickets work? I pulled from main.

File: gulpfile.js (hunk 4 above)
Comment (msandhuNHS) [reply gulpfile.js:15]: Will leave commented out, doesn't effect anything.

Comment (zainm97) [reply gulpfile.js:15]: common dev processes and standards need to be followed. Please address

File: src/app/components/pagination.js (hunk 2 above)
Comment (zainm97) [reply src/app/components/pagination.js:45]: Please can you address the comment

File: src/app/components/pagination.js (hunk 3 above)
Comment (zainm97) [reply src/app/components/pagination.js:49]: Please can you address the comment

File: src/app/views/clinic_information/ClinicInformation.js (hunk 5 above)
Comment (zainm97) [reply src/app/views/clinic_information/ClinicInformation.js:325]: Please can you address the comment before closing it out

File: src/app/views/invitation_planning/InvitationPlanning.js (hunk 6 above)
Comment (zainm97) [reply src/app/views/invitation_planning/InvitationPlanning.js:52]: Please may you handle the response. Seems best practice to put something in even if its just a log

File: src/app/views/invitation_planning/InvitationPlanning.js [hunk 7]
```diff
@@ -61,7 +61,7 @@ class InvitationPlanning extends Component {
         { quintiles: updatedQuintile }
       )
       .then((response) => {
-        console.log("response -> " + response.status);
+
```
[the remark this answers is not in the document]
Comment (zainm97) [reply src/app/views/invitation_planning/InvitationPlanning.js:64]: Please may you handle the response. Seems best practice to put something in even if its just a log

**Review by zainm97 — CHANGES_REQUESTED** [review]
Hi Manny, none of my comments were address - they were just marked as closed with no comments back. Please can you address the conversation at least

File: src/app/components/pagination.js (hunk 2 above)
Comment (msandhuNHS) [reply src/app/components/pagination.js:45]: This is for debugging purposes, am commenting out so main does not have logs

File: src/app/views/clinic_information/ClinicInformation.js (hunk 5 above)
Comment (msandhuNHS) [reply src/app/views/clinic_information/ClinicInformation.js:325]: Have just added two fields. As per Cheng's comment, we need to get it functioning, it's perfectly fine to treat it this way for now. Code cleanup if necessary can occur later. I am happy with this code.

File: src/app/views/invitation_planning/InvitationPlanning.js (hunk 6 above)
Comment (msandhuNHS) [reply src/app/views/invitation_planning/InvitationPlanning.js:52]: Errors should never log to browser unless for debugging in dev phase. This is fine to comment out. Again all logs are commented out for main. They can be enabled during development.

File: src/app/views/invitation_planning/InvitationPlanning.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (msandhuNHS) [reply src/app/views/invitation_planning/InvitationPlanning.js:64]: same as before.

File: gulpfile.js (hunk 4 above)
Comment (msandhuNHS) [reply gulpfile.js:15]: Yes, going into main, leave commented out, only needed for debug purposes.

Comment (msandhuNHS) [reply gulpfile.js:15]: Have now commented back in.

File: src/app/components/pagination.js (hunk 2 above)
Comment (msandhuNHS) [reply src/app/components/pagination.js:45]: Have now deleted these logs as per Cheng's request.

File: src/app/components/pagination.js (hunk 3 above)
Comment (msandhuNHS) [reply src/app/components/pagination.js:49]: Deleted as above.

File: src/app/views/clinic_information/ClinicInformation.js (hunk 5 above)
Comment (msandhuNHS) [reply src/app/views/clinic_information/ClinicInformation.js:325]: Other than directly unpacking the request response into setState() I am unsure how to clean this up. I'd prefer having consts hold the response data.

File: src/app/views/clinic_information/ClinicInformation.js [hunk 8]
```diff
@@ -325,29 +327,36 @@ class ClinicInformation extends Component {
...
-                    this.setState({
-                      targetFillToInputValue: targetPercentageValue,
-                      appsToFill: Math.floor(this.state.recentInvitationHistory.appsRemaining * (targetPercentageValue / 100)),
-                    });
-                  });
+                this.setState({
+                  appsToFill: Math.floor(this.state.recentInvitationHistory.appsRemaining * (this.state.targetFillToInputValue / 100)),
+                });
               }
+              // () => {
```
Comment (zainm97) [inline src/app/views/clinic_information/ClinicInformation.js:338]: why is this commented out? it should be removed or put in a helper file if we are going to use it in the future. Or at the very least it should have a comment explaining why it is commented out

File: src/app/views/clinic_information/LsoaTable.jsx [hunk 9]
```diff
@@ -1,52 +1,59 @@
...
       </div>
       <div class="govuk-input__wrapper" id="distance">
         <select
-          class={
-            "nhsuk-select"
-          }
+          class={"nhsuk-select"}
           id="milesFromSite"
-          name="miles"
+          name="miles"milesOptions
```
Comment (zainm97) [inline src/app/views/clinic_information/LsoaTable.jsx:35]: odd, what is this doing

Comment (msandhuNHS) [reply src/app/views/clinic_information/LsoaTable.jsx:35]: Typo have removed. Didn't break functionality.

File: src/app/views/clinic_information/ClinicInformation.js (hunk 8 above)
Comment (msandhuNHS) [reply src/app/views/clinic_information/ClinicInformation.js:338]: I'm not too sure how this fits into the functionality as it was implemented from a different ticket. Commented out incase someone else needed it. Thought it self explanatory so didn't add a comment. I can delete it as I don't think it's needed, certainly doesn't break new functionality.

Comment (zainm97) [reply src/app/views/clinic_information/ClinicInformation.js]: I think i was trying to say that this functionality will be used later in the calculate number to invite button so it is ok to leave in but a small comment describing the above will be helpful to whoever picks up the ticket

**Pull request closed without merging**